apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx

```diff
@@ -28,5 +35,27 @@ export const FirebaseFeatureFlagsProvider = ({ children }: Props): JSX.Element =
     }
   };
 
-  return <FeatureFlagsProvider getFeature={getFeature}>{children}</FeatureFlagsProvider>;
+  const overrideFeature = (key: FeatureId, value: Feature): void => {
+    const actualRemoteValue = getFeature(key, false);
+    if (!isEqual(actualRemoteValue, value)) {
+      const overridenValue = { ...value, overridesRemote: true };
+      setLocalOverrides(currentOverrides => ({ ...currentOverrides, [key]: overridenValue }));
+    } else {
+      console.error("Not overriding");
+    }
+  };
+
+  const resetFeature = (key: FeatureId): void => {
+    setLocalOverrides(currentOverrides => ({ ...currentOverrides, [key]: undefined }));
+  };
```

Comment:
Should we use `useCallback` for every method passed to `FeatureFlagsProvider` ?

Reply:
Not for reset since it has no dependencies that change, for override it made sense

Reply:
`useCallback` is more about keeping a constant ref of the function for every re-renders